functionaltests/src/test/java/specs/paymentmethods/sofortueberweisung/ChargeImmediatelyFixture.java

```diff
@@ -154,7 +154,7 @@ public boolean executeRedirectForPayments(final String paymentNames) throws Exec
 
         // run all 3 payments approval in parallel, aka 3 different sessions
         // and collect successfully approved redirect URLs
```

Comment:
You have to change comment accordingly and please also explain in the comment why it is not in parallel anymore.

Reply:
I tried to put back `parallel`, but the tests fails, so this fix is correct. @ahalberkamp pls have a look and answer Roman's comment.

Reply:
@lojzatran yes fix is as described in PR description to reduce parallel execution but my request was to just align the java code comment to the change.

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/230/commits/f83ef63d7885699b6a7555d3767470734977171e